spaceship/config.py

```diff
@@ -4,6 +4,8 @@
 
 class Config:
   IN_PRODUCTION = bool(os.environ.get('IN_PRODUCTION', False))
```

Comment:
May be clearer to just call this USE_WORK_QUEUE.

Reply:
good call! will do.